Docker-Swarm-deployment/single-node/elk/README.md

```diff
@@ -16,34 +16,42 @@ Docker swarm stack for Elasticsearch-logstash-Kibana (ELK) Deployment.
 ```
 ./generate-keystore.sh
 ```
-4. Create S3 bucket and corresponding IAM user with programmatic access to only that bucket. Copy the access-key id and access-secret key to ``secrets/pki/s3-access-key`` and ``secrets/pki/s3-secret-key``
-5. Secrets directory after generation of secrets
+4. Secrets directory after generation of secrets
 
-## Required secrets
 ```sh
 secrets/
 ├── keystores
-│   ├── elasticsearch.keystore
 │   ├── kibana.keystore
 │   └── logstash.keystore
 ├── passwords
 │   ├── elasticsearch-cat-password
 │   ├── elasticsearch-fs-password
 │   ├── elasticsearch-rs-password
 │   ├── elasticsearch-su-password
+│   ├── es-password.env
```

Comment:
what/how to define this env file? can u put an example file in example-secrets/secrets/passwords folder?

Reply:
It is generated by the `create-secrets.sh` script, the user does not need to configure it manually.